test/unit/packwerk/parse_run_test.rb

```diff
@@ -376,7 +376,7 @@ def something
       expected_message = <<~EOS
         No offenses detected
         No stale violations detected
-        components/source cannot have dependency violations on components/destination because strict mode is enabled for dependency violations in components/source/package.yml
+        components/source cannot have dependency violations on components/destination because strict mode is enabled for dependency violations in the enforcing package's package.yml
```

Comment:
Should we add a test for the other case?

Reply:
I started adding a test but I think it's okay without, since the other case would just involve other test setup, but the same assertions and code paths. If there was a regression, it would show up equally in another test vs. the current one, since whether a checker's enforcement is on the referencing or consuming package is something that isn't part of the interface quite yet (just a conceptual consideration).